velox/serializers/PrestoSerializer.cpp

```diff
@@ -294,6 +294,47 @@ void readValues<IntervalDayTime>(
   }
 }
 
+static const int64_t kInt64DeserializeMask = ~(static_cast<int64_t>(1) << 63);
```

Comment:
nit: static constexpr ... can you just move this to readUnscaledLongDecimal? or move to the anonymous namespace at the head of this file? Thanks!